docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -962,13 +956,90 @@ app.get('/escrows', async (req, res) => {
 });
 ```
 
+### Deployment {#deployment}
+
+Now that you have an indexer and an API service, you can deploy your move package and start the indexer and API service.
+
+1. Install dependencies by running
```

Comment:
```suggestion
1. Install dependencies by running `pnpm install --ignore-workspace`.
```
I think we can use inline code for these. That way we can also use ending punctuation to better align with other lists on the page. And we could also include yarn if wanted:
```suggestion
1. Install dependencies by running `pnpm install --ignore-workspace` or `yarn install --ignore-workspace`.
```